### PR TITLE
update spec

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

**Comment by yuval-k** [comment]
spec change suggestion - implementation will follow and a follow-up PR.

**Review by ilackarms — APPROVED** [review]

File: spec/README.md [hunk 1]
```diff
@@ -63,18 +63,18 @@ Control planes then configure the Envoy instances to load the filter via the loc
...
 
 | Media Type | Type | Description |
 |------------|------|-------------|
-| application/vnd.module.wasm.config.v1+json | JSON Object | Configuration for the Target WASM runtime.
-| application/vnd.module.wasm.content.layer.v1+wasm | binary data (byte array) | The compiled module data |
+| application/vnd.io.solo.wasm.config.v1+json | JSON Object | Configuration for the Target WASM runtime.
+| application/vnd.io.solo.wasm.content.layer.v1+wasm | binary data (byte array) | The compiled module data |
 
-`application/vnd.module.wasm.config.v1+json` Property Descriptions:
+`application/vnd.io.solo.wasm.config.v1+json` Property Descriptions:
```
Comment (mandarjog) [inline spec/README.md:77]: Are you thinking that this is "solo" format? And we will have vnd.google or vnd.istio once we adopt this?

Alternatively we can have vnd.cncf.wasm or similar.

Comment (yuval-k) [reply spec/README.md:77]: i'm flexible on this; but didn't want to use a cncf prefix without discussing with the cncf first.
changed it solo based on advice from the OCI team. I think once this stabilizes and get industry agreement we can change to vnd.cncf.
WDYT?

Comment (yuval-k) [reply spec/README.md:77]: so to summarize, I don't see it as a solo format; This change is to be similar to other [oci specs](https://github.com/opencontainers/artifacts/blob/master/artifact-authors.md) and we would be happy to change this to something more appropriate

Comment (yuval-k) [reply spec/README.md:77]: opened https://github.com/solo-io/wasm/pull/192 to address this

Comment (mandarjog) [reply spec/README.md:77]: Ok. I think we should talk to cncf soon, it is best that the tools use the right media type early. The spec can continue to evolve.

**Pull request merged**